Gemfile.lock

```diff
@@ -461,4 +462,4 @@ RUBY VERSION
    ruby 2.7.1p83
 
 BUNDLED WITH
-   2.1.4
+   2.2.8
```

Comment:
Does this mean I have an incorrect version of something installed that I should change?

Reply:
It looks like you have the latest version of bundle on your system and we were all on 2.1.4

Reply:
We can all upgrade to 2.2.8 I suppose